src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -143,7 +145,7 @@ static bool InitDLL(HANDLE hPalInstance)
         return false;
 #endif
 
-#ifdef TARGET_UNIX
+#ifdef FEATURE_RANDOMIZED_GS_COOKIES
```

Comment:
Instead of disabling randomization, it would be better to keep the randomization and just not place it into read-only section.

Non-randomized GS cookie is not very useful since it has 100% predicable value. It is better to give up on the read-only nature than on randomization.

Reply:
Makes sense. So if I change this to be `FEATURE_READONLY_GS_COOKIES`, would it be sufficient to remove the `volatile READONLY_ATTR const` part of the definition if the feature is not enabled? Or would it also need to be placed in a specific section?

Reply:
> FEATURE_READONLY_GS_COOKIES

Nit: `FEATURE_READONLY_GS_COOKIE`. There is only one.